.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -46,7 +46,7 @@ If there are following updates in the PR, ensure to request an approval from API
 - [ ] Removing properties in stable version
 - [ ] Removing API version(s) in stable version
 - [ ] Updating API in stable version with Breaking Change Validation errors
-- [ ] Updating API(s) in preview over 1 year
+- [ ] When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. (Ensure your swagger merge right after your service published)
```

Comment:
add (Ensure...) not here but in above changelog section 
also there may be more updates in this PR template just synced through teams